bundles/org.openhab.binding.teslascope/README.md

```diff
@@ -0,0 +1,241 @@
+| sunroof                     | Dimmer                   | Sunroof                       | Indicates the opening state of the sunroof (0% closed, 100% fully open)                                          |
+| time-to-full-charge         | Number                   | Time To Full Charge           | Number of hours to fully charge the battery                                                                      |
+| tpms-pressure-fl            | Number:Pressure          | Tyre Pressure FL              | Tyre Pressure Front Left in Bar                                                                                  |
+| tpms-pressure-fr            | Number:Pressure          | Tyre Pressure FR              | Tyre Pressure Front Right in Bar                                                                                 |
+| tpms-pressure-rl            | Number:Pressure          | Tyre Pressure RL              | Tyre Pressure Rear Left in Bar                                                                                   |
+| tpms-pressure-rr            | Number:Pressure          | Tyre Pressure RR              | Tyre Pressure Rear Right in Bar                                                                                  |
+| tpms-soft-warning-fl        | Switch                   | Tyre Pressure Soft Warning FL | Tyre Pressure Soft Warning Front Left                                                                            |
+| tpms-soft-warning-fr        | Switch                   | Tyre Pressure Soft Warning FR | Tyre Pressure Soft Warning Front Right                                                                           |
+| tpms-soft-warning-rl        | Switch                   | Tyre Pressure Soft Warning RL | Tyre Pressure Soft Warning Rear Left                                                                             |
+| tpms-soft-warning-rr        | Switch                   | Tyre Pressure Soft Warning RR | Tyre Pressure Soft Warning Rear Right                                                                            |
+| usable-battery-level        | Number                   | Usable Battery Level          | Indicates the % of battery that can be used for vehicle functions like driving                                   |
+| valet-mode                  | Switch                   | Valet Mode                    | Enable or disable Valet Mode                                                                                     |
+| wiper-blade-heater          | Switch                   | Wiperblade Heater             | Indicates if the wiperblade heater is switched on                                                                |
+
+## Full Example
+
+demo.Things:
+
+```java
+teslascope:service:model3 [ apiKey="xxxx", publicID="aXb3" ] {
+}
```

Comment:
```suggestion
teslascope:service:model3 [ apiKey="xxxx", publicID="aXb3" ] 
```

also add the code fence close tagf (somehow i can't suggest that here.

Reply:
Fixed.